opamp/observiq/observiq_client.go

```diff
@@ -240,7 +242,15 @@ func (c *Client) onConnectHandler() {
 			c.logger.Error("OpAMP client failed to set package statuses", zap.Error(err))
 		}
 	} else {
+		c.logger.Error(
```

Comment:
For zap logging we can include attributes by using `zap.String("package_name", "packagestate.CollectorPackageName")` and it'll add them to the Json. I think this might be better than doing string interpolation. The main body message can just say `Package update failed because...`

Reply:
Same comment with all the logs added here.

Reply:
@cpheps ah cool. i'll do that